src/test/plugin.test.ts

```diff
@@ -176,6 +176,26 @@ describe('AsyncAPI EventCatalog Plugin', () => {
         );
       });
 
+      it('when the AsyncAPI service is already defined in EventCatalog and the versions match, folderName is considered', async () => {
+        // Create a service with the same name and version as the AsyncAPI file for testing
+        const { writeService } = utils(catalogDir);
+        const existingVersion = '1.0.0';
+        await writeService(
+          {
+            id: 'account-service',
+            version: existingVersion,
+            name: 'Custom folderName',
+            markdown: 'This service is stored in a folder with custom name',
+            specifications: { openapiPath: 'simple.asyncapi.yml' },
+          },
+          { path: 'my-custom-folder' }
+        );
+
+        await plugin(config, {
```

Comment:
Should we assert something here? Making sure the old thing is versioned as an example? Or the path to the old version includes this folder name?

Reply:
I understand you want an assertion here. The only thing I can think about is check if the file has been updated/overwritten. We could check the datetime at filesystem level. We need to add a sleep in the test in case. WDYT ? 

Yet another option is to add a property to the schema that allow me to understand when was the file modified and use it in the assertion. This can also be useful for debugging or to let ppl know how "fresh" the specs are.

Reply:
Yeah no problem. I just wanted to make sure that the folder name is taken into consideration. So I just added a simple assert to make sure the folder is there with the markdown as we expect. 

Thanks for raising the PR!